plugins/discovery-azure/src/test/java/org/elasticsearch/cloud/azure/AbstractAzureComputeServiceTestCase.java

```diff
@@ -48,7 +48,7 @@ protected Settings nodeSettings(int nodeOrdinal) {
 
         // We add a fake subscription_id to start mock compute service
         builder.put(Management.SUBSCRIPTION_ID, "fake")
-            .put(Discovery.REFRESH, "5s")
+            .put(Discovery.REFRESH_SETTING.getKey(), "5s")
```

Comment:
you understand it correctly. Discovery.REFRESH_SETTING.getKey() yields the string discovery.azure.refresh_interval